cluster/v1beta1/types_placement.go

```diff
@@ -94,6 +94,42 @@ type PlacementSpec struct {
 	// certain taints to be selected by placements with matching tolerations.
 	// +optional
 	Tolerations []Toleration `json:"tolerations,omitempty"`
+
+	// Decision Strategy divide the created placement decision to groups and define number of clusters per decision group.
+	// +optional
+	DecisionStrategy DecisionStrategy `json:"decisionStrategy,omitempty"`
+}
+
+// Subset of the created placementDecision with label.
+type DecisionGroup struct {
+	// Label to be added to the created placement Decisions
+	// +kubebuilder:validation:Required
+	// +required
+	Label string `json:"labels,omitempty"`
+
+	// LabelSelector to select clusters subset by label.
+	// +kubebuilder:validation:Required
+	// +required
+	ClusterSelector ClusterSelector `json:"groupClusterSelector,omitempty"`
+}
+
+// Group the created placementDecision into decision groups based on the number of clusters per decision group.
+type DecisionStrategy struct {
+	// Define subsets of the created placement decisions and add labels to the placementDecision.
+	// +optional
+	DecisionGroups []DecisionGroup `json:"decisionGroups,omitempty"`
+
+	// ClusterPercentagePerDecisionGroup is a percentage of the total selected clusters that will divide the clusters to groups.
+	// ex; for a total 100 clusters selected, ClusterPercentagePerDecisionGroup equal to 20% will divide the placement decision to 5 groups each group should have 20 clusters.
+	// +kubebuilder:validation:Enum=None;"5%";"10%";"15%";"20%";"25%";"30%";"50%";"100%"
+	// +kubebuilder:default:=None
+	// +optional
+	ClusterPercentagePerDecisionGroup string `json:"clusterPercentagePerDecisionGroup,omitempty"`
```

Comment:
It is actually MaxCountPerGroup right? I still think we should only have one field that can be set as IntOrString